docs/updating/from_1.x_2.x/update_db_to_2.5.md

```diff
@@ -890,6 +890,8 @@ if (req.restarts == 0 && resp.status == 301 && req.http.x-fos-original-url) {
 }
 ```
 
+If you want to provide custom configuration in VCL snippets, for example, to handle exceptions or special cases, make sure that in your `ez_main.vcl` file you move the `#FASTLY recv` macro call to [a new location](https://github.com/ezsystems/ezplatform-http-cache-fastly/blob/a1f193438f1d6cf604d331630f14f91a7f42dd77/fastly/ez_main.vcl#L25), right after the `Preserve X-Forwarded-For in all requests` section.
```

Comment:
```suggestion
If you want to provide custom configuration in VCL snippets, for example, to handle exceptions or special cases,
make sure that in your `ez_main.vcl` file you move the `#FASTLY recv` macro call to [a new location](https://github.com/ezsystems/ezplatform-http-cache-fastly/blob/a1f193438f1d6cf604d331630f14f91a7f42dd77/fastly/ez_main.vcl#L25),
right after the `Preserve X-Forwarded-For in all requests` section.
```
On the topic of snippets, how about extracting this paragraph to a snippet in https://github.com/ezsystems/developer-documentation/tree/master/docs/snippets?

Reply:
We can not link to https://github.com/ezsystems/ezplatform-http-cache-fastly/blob/a1f193438f1d6cf604d331630f14f91a7f42dd77/fastly/ez_main.vcl because that is a private repo.

What about saying that in this release there is a new Fastly .vcl so make sure you deploy the new one on Fastly. The updated vcl is located in `vendor/ezsystems/ezplatform-http-cache-fastly/fastly/ez_main.vcl

Reply:
Please see it after my changes

Reply:
Looks good now